NOTICE.txt

```diff
@@ -1,7 +1,15 @@
-OpenSearch
-Copyright 2021 OpenSearch Contributors
-This product includes software, including Kibana source code, developed by Elasticsearch (http://www.elastic.co).
-Copyright 2012-2021 Elasticsearch B.V.
+OpenSearch (https://opensearch.org/)
+Copyright OpenSearch Contributors
+
+This product includes software, including Kibana source code,
+developed by Elasticsearch (http://www.elastic.co).
+Copyright 2009-2021 Elasticsearch B.V.
```

Comment:
OK, I'm still not convinced we have the dates correct - we're going from `2012-2021` to `2009-2021`.  We fixed the end date in https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3308/files, but I don't expect either of these years to change.

Reply:
@tmarkley Can you clarify?

Reply:
The new value is correct. It should not change in theory but the purpose of the date is to indicate the start and end year when the code was first created, and there is definitely an instance that mentions that the code was created in 2009 so updating this is correct.